product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/02_connecting_from_aws/index.mdx

```diff
@@ -116,14 +162,15 @@ Now that your endpoint service is created, you can connect it to the cluster VPC
     psql -h vpce-XXXXXXXXXXXXXXXXXXXX.eu-west-1.vpce.amazonaws.com -U edb_admin -p 5432     
     __OUTPUT__
     Password for user edb_admin: 
-
-    psql (13.4 (Ubuntu 13.4-1.pgdg28.84+1), server 13.4.8 (Debian 13.4.8-1+deb10)) 
-    WARNING : psql major version 13, server major version 13. Some psql features might not work. 
-    SSL connection (protocol : TLSV1.3cipherTLS_AES_256_GCM_SHA384, bits : 256, compression : off) Type "help" for help. 
+    ```
```

Comment:
Again, unclear to me what we're attempting to present here.

Reply:
Same answer as [this thread](https://github.com/EnterpriseDB/docs/pull/4160/files#r1214494106) also apply here

Reply:
So we want to remove the following, correct?
```shell     
    # Replace the port 5432 below if that has been changed in the Big Animal DB Configuration
    psql -h vpce-XXXXXXXXXXXXXXXXXXXX.eu-west-1.vpce.amazonaws.com -U edb_admin -p 5432     
    __OUTPUT__
    Password for user edb_admin: 
    ```
    ```
    psql (15.2)
    SSL connection (protocol: TLSv1.3, cipher: TLS_AES_256_GCM_SHA384, compression: off)
    Type "help" for help.

    edb_admin=>
    ```